src/lib.js

```diff
@@ -139,7 +141,7 @@ const skip = S.map(R.evolve({ snippets: S.filter(matchNoSkip) }))
 // @link: https://github.com/isaacs/node-glob#glob-primer
 
 // taskOfSnippets :: Object -> Object -> Int -> GlobPattern -> Future([FileResult])
-const taskOfSnippets = (dependencies, globals, timeout, glob) =>
+const taskOfSnippets = ({ dependencies, globals, timeout }, glob) =>
```

Comment:
Shouldn't `globals` be `config`?

Reply:
The whole object is config, I'm just destructing.

Reply:
Yeah it should, don't know why I added that instead of installing. Will fix

Reply:
Ohhh, you sleazy destructurer.